File: app/src/main/java/org/breezyweather/daily/adapter/model/Margin.java
package org.breezyweather.daily.adapter.model;

import org.breezyweather.daily.adapter.DailyWeatherAdapter;

public class Margin implements DailyWeatherAdapter.ViewModel {

    public static boolean isCode(int code) {
        return code == -2;
    }

    @Override
    public int getCode() {
        return -2;
    }
}

File: app/src/main/java/org/breezyweather/daily/adapter/model/Margin.kt
package org.breezyweather.daily.adapter.model

import org.breezyweather.daily.adapter.DailyWeatherAdapter

class Margin : DailyWeatherAdapter.ViewModel {
    override val code = -2

    companion object {
        fun isCode(code: Int) = (code == -2)
    }
}
